kolibri/plugins/setup_wizard/assets/src/views/importFacility/SelectSuperAdminAccountForm.vue

```diff
@@ -138,6 +145,9 @@
         this.shouldValidate = false;
         if (this.$refs.password) {
           this.$refs.password.resetAndFocus();
+          if (this.selected.label === this.facility.username) {
```

Comment:
I'm guessing that this is whereabouts this issue would be found in the code.

I get here and the username is defaulted to the username I used in the first place (`learningequality`) to start import and the password was autofilled.

If I go from `learningequality` (I'm testing using the beta server) to `micah` and then go back to `learningequality` - the password field is still filled with the `learningequality` password.

But - if you go:

`learningequality` -> `Create new super admin` -> `learningequality` --- then the password is no longer filled out. If I then go from there to `micah` and back to `learningequality` then the password is filled again.

Seems that this `resetFormAndRefocus()` method isn't being called whenever you switch away from the Create a new super admin form.